0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
 # Interledger Architecture
 
-This document outlines the Interledger architecture and explains how the different layers relate to each other. The [Interledger Protocol (ILP)](#interledger-protocol-ilp) is one layer in the Interledger architecture.
+The Interledger protocol suite provides for secure payments across multiple assets on different ledgers.
 
-The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
+The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite.
```

Comment:
As an outsider coming to this - I agree.  "ILP" seems to be the widely used term to reference the entire suite/stack. It's certainly the term used by everyone I know outside the dev circle who discusses, well... ILP.  The current situation that one layer of the stack also shares the same name as the whole stack is confusing to newcomers and a hinderance to unambiguous discussion.  The ILPC distinction suggested here is a good solution, I would take it further and drop the "protocol".  We know it's a protocol already given the suite would be/is referred to as the Interledger Protocol, so we could simply refer to that layer as Interledger Core.

So, the Interledger Protocol (ILP) is the entire stack of protocols, one layer of which is Interledger Core, or ILC.